### ISSUE TITLE
The word "bibliography" in egXMLEndHook is hard-coded, but should be an i18n link

### ISSUE THREAD

**Issue by martindholmes** [TEIC/Stylesheets#478]
Stylesheets group discovered this while working on ticket #471. The location is odds/guidelines.xsl line 776.

**Comment by sydb** [comment]
And at first glance it looks like the key that it should be i18ned to is “biblioWords”.

I, for one, do not have a strong opinion about whether or not that is an appropriate key. But its translations look to me (who speaks no language other than English)-: like the right stuff, and it is never used.

**Comment by sydb** [comment]
I note also that the jTEI profile in some ways does this better than the Stylesheets per se, and in one way perhaps worse. They include an attribute on each <text> for a plural value, and consistently use keys that make it clear what is going on (they all end in “-label” except for “and”). Those are good ideas to emulate. They also put the key on `@xml:id`. I am not sure what I think of that. The syntactic constraints on an ID value are not likely to be problematic, and the uniqueness constraint in some circumstances would be quite useful. But the syntactic constraints are not entirely benign (think if you wanted to translate “2D” and “3D”), and one can imagine circumstances where the uniqueness constraint gets in the way.

Anyway, in a8850ef I have changed
~~~xml
<xsl:text>bibliography</xsl:text>
~~~
to
~~~xml
<xsl:sequence select="lower-case( tei:i18n('biblioWords') )"/>
~~~
in all 3 places it occured (all in odds/guidelines.xsl).

Both `make test` and `cd Test2/; ant test` pass (except for LaTeX) on my local system; Guidelines built using this change look right in that “bibliography” appears in the lower R corner of most examples. I also checked a couple of other languages, and although I can’t _read_ the result, it looks reasonable.
